WordPress-iOS-Shared/Core/WPImageSource.m

```diff
@@ -24,17 +25,19 @@ + (instancetype)sharedSource
 
 - (void)dealloc
 {
-    [_downloadingQueue cancelAllOperations];
+    [_downloadsSession invalidateAndCancel];
 }
 
 - (id)init
 {
     self = [super init];
     if (self) {
-        _downloadingQueue = [[NSOperationQueue alloc] init];
         _urlDownloadsInProgress = [[NSMutableSet alloc] init];
         _successBlocks = [[NSMutableDictionary alloc] init];
         _failureBlocks = [[NSMutableDictionary alloc] init];
+
+        NSURLSessionConfiguration *configuration = [NSURLSessionConfiguration defaultSessionConfiguration];
+        _downloadsSession = [NSURLSession sessionWithConfiguration:configuration];
```

Comment:
Looking at the current use cases, I'd say we don't want background downloads as the images being downloaded aren't important enough. If that becomes useful, the caller should be able to inject the session/session configuration, but that's something to revisit when we use NSURLSession across the app.